frontend/src/common/config.ts

```diff
@@ -1,6 +1,6 @@
 import AxiosStatic from 'axios';
 
-export const frontendBaseURL = `${location.protocol}//${location.host}${import.meta.env.BASE_URL}`;
```

Comment:
While i like the change from "one needs to know from context" to something explicit, I cannot say if this is correct. @tobihagemann Can we just replace BASE_URL with a fixed path?

Reply:
What this PR doesn't show: I did half a day of experiments with `BASE_URL` and decided to ditch it, as it is not applicable to our app at all. At first I bound the router to the BASE_URL and set the BASE_URL in `vite.config.ts`.

This leads to wrong asset paths, though. We want vite to generate everything at `/` but the router to put everything under `/app`.

Reply:
Btw it was me [who introduced the `BASE_URL`](https://github.com/cryptomator/hub/commit/52b1035259495926fbd66492caec2dbb47d4b763#diff-8d302fe8581dca01e60aaf346dfdb0f8e24731551e7e5f91e5da9fc64250f7d2R37), thinking that we might need to customize the apps root. Back then I didn't know what effect this had on the build.

Reply:
Ok, than this change is legit.